tests/rules/test_hadoop_dfs_missing_dash.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from thefuck.rules.hadoop_dfs_missing_dash import match, get_new_command
+from tests.utils import Command
+
+
+@pytest.mark.parametrize('command', [
+    Command(script='./bin/hdfs dfs ls', stderr='ls: Unknown command\nDid you mean -ls?  This command begins with a dash.'),
```

Comment:
The previous solution handled any hdfs dfs <blah> already as it blinded adds a "-" to the second param.

But I've now pushed a version that ignores the command and just looks at the error message.